### PR TITLE
Prepare for new release

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
Many thanks to contributing to nf-core/mhcquant!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If necessary, also make a PR on the [nf-core/mhcquant branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/mhcquant)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/mhcquant/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,75 +1,114 @@
...
+### `Fixed`
+
+- version upgrade mhcnuggets and mhcflurry
+- output file order in intermediate results
+- increased run times for MS search and variant translation
+- RT prediction
+- Quantification FDR
+- Variant pass filter
+
+## dev
```
Comment (Zethson) [inline CHANGELOG.md]: We can remvoe this dev and just merge it with 1.4 now :)

File: bin/vcf_reader.py [hunk 2]
```diff
@@ -0,0 +1,1180 @@
...
+
+__author__ = 'mohr, walzer'
+VERSION = "1.1"
+
+ID_SYSTEM_USED = EIdentifierTypes.ENSEMBL
+transcriptProteinMap = {}
+transcriptSwissProtMap = {}
+
+# Adapt depending on system
+REF_PATH = ['/lustre_cfc/qbic/reference_genomes/IRMA_references/uniprot_all_swissprot_UP000005640.fasta', '/lustre_cfc/qbic/reference_genomes/IRMA_references/Homo_sapiens.GRCh38.pep.all.fa']
```
Comment (Zethson) [inline bin/vcf_reader.py]: How is this supposed to work?
People are not supposed to edit files.

Comment (Leon-Bichmann) [reply bin/vcf_reader.py]: oh right, yes that we should take out

File: bin/vcf_reader.py [hunk 3]
```diff
@@ -0,0 +1,1180 @@
...
+
+# List of coding genes
+GENE_LIST = '/lustre_cfc/qbic/reference_genomes/all_coding_genes_GRCh_ensembl_hgnc.tsv'
```
Comment (Leon-Bichmann) [inline bin/vcf_reader.py]: this one as well

File: CHANGELOG.md (hunk 1 above)
Comment (Leon-Bichmann) [reply CHANGELOG.md]: Ok, will do this in a later PR

**Comment by Leon-Bichmann** [comment]
Anyone has an idea why the 3 tests are failing?

**Comment by Zethson** [comment]
usage.md:288:1 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]

**Comment by Leon-Bichmann** [comment]
ok, but what about this one:
```
Run { [[ $(git remote get-url origin) == *nf-core/mhcquant ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
  { [[ $(git remote get-url origin) == *nf-core/mhcquant ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
  shell: /bin/bash -e {0}
fatal: not a git repository (or any of the parent directories): .git
```

**Comment by Zethson** [comment]
No idea. Maybe @drpatelh knows?

File: CHANGELOG.md (hunk 1 above)
Comment (Leon-Bichmann) [reply CHANGELOG.md]: done

**Comment by Zethson** [comment]
@Leon-Bichmann 
Latest version in the template should fix this:
https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/.github/workflows/branch.yml

Since a release is some time of you can copy that code (replace cookiecutter annotations of course).

**Comment by Leon-Bichmann** [comment]
Ok ill merge this later today if there are no more requested reviews

**Pull request merged**